client/js/lounge.js

```diff
@@ -1,4 +1,18 @@
-"use strict";
+// vendor libraries
+import "jquery-ui/ui/widgets/sortable";
+import $ from "jquery";
+import io from "socket.io-client";
```

Comment:
At the moment, this does build (at least on my machine), and I'm very confused why it builds on Travis CI (at least for the version of Node/npm we have in common).

I get this at the end of `npm install` on a clean repo (`git clone https://github.com/thelounge/lounge && cd lounge && git checkout pr/640 && npm i`):

```js
   [0] multi vendor 88 bytes {1} [built] [1 error]
    + 75 hidden modules

ERROR in multi vendor
Module not found: Error: Cannot resolve module 'socket.io-client' in /private/tmp/lounge
 @ multi vendor

ERROR in ./client/js/lounge.js
Module not found: Error: Cannot resolve module 'socket.io-client' in /private/tmp/lounge/client/js
 @ ./client/js/lounge.js 9:14-41
```

App does not run in the browser, I get this in the browser console:

```js
 bundle.vendor.js:105 Uncaught Error: Cannot find module "socket.io-client"
    at bundle.vendor.js:105
    at Object.<anonymous> (bundle.vendor.js:105)
    at t (bootstrap 0a4ff42…:50)
    at bootstrap 0a4ff42…:93
    at bootstrap 0a4ff42…:93
lounge.js:4 Uncaught Error: Cannot find module "socket.io-client"
    at lounge.js:4
    at Object.<anonymous> (lounge.js:4)
    at t (bootstrap 0a4ff42…:50)
    at window.webpackJsonp (bootstrap 0a4ff42…:21)
    at bundle.js:1
```

I replaced this line with `import io from "socket.io/lib/client";` to fix it.
An alternative is to add `socket.io-client` in `package.json` but I'd rather count on the version provided by the server. I'll add this in an extra commit to see if Travis is happy with it, let me know what you think.